test/functional/mempool_limit.py

```diff
@@ -20,55 +23,55 @@ def set_test_params(self):
         ]]
         self.supports_cli = False
 
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
-
     def run_test(self):
-        txouts = gen_return_txouts()
-        relayfee = self.nodes[0].getnetworkinfo()['relayfee']
+        node = self.nodes[0]
```

Comment:
It is a bit confusing to review if the `self.node[0]` -> `node` replacement is done in the same commit as the miniwallet changes. Maybe split it up?

Reply:
Thank you for the suggestion. I have split up the commits